pkg/sloop/server/internal/testFiles/testconfig.yaml

```diff
@@ -1,43 +1,14 @@
 ---
 ConfigFile: ''
 leftBarLinks: 
-resourceLinks: 
```

Comment:
Why are we removing these values from the test configs?

Reply:
to check if it works the way we want with an incomplete config file input.

Reply:
gotcha, thanks for the info 👍